### PR TITLE
Docs: README

### PR DESCRIPTION
**Pull request by DirtyF**
👋 Hey team,

I went through the starter and revamped the README to make it easier to follow.

[👀 View README](https://github.com/DirtyF/tina-cloud-starter/blob/docs/README/README.md)

- [x] Starter Scope
- [x] Starter Preview
- [x] Table of Contents
- [x] Reorganize and proof-read content
- [x] Cleanup used images to make the repository lighter (-5Mo)

Hope it helps 🤞

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -146,11 +220,11 @@ sections:
       - post
 ```
 
-And I have a document located at `content/posts/hello-world.md`, you'll find that document at `http://localhost:3000/posts/hello-world`.
+A document is located at `content/posts/hello-world.md`, lives at `http://localhost:3000/posts/hello-world`.
```
Comment (jamespohalloran) [inline readme.md:232]: I don't know that "located at" vs "lives at" makes sense to me

File: readme.md [hunk 2]
```diff
@@ -146,11 +220,11 @@ sections:
...
 
-> Notice that `hello-world` is considered the **relative** path here, we don't need to specify `content/posts/hello-world` because that's been configured in our `settings.yml`.
+Notice that `hello-world` is considered as the **relative** path, we don't need to specify `content/posts/hello-world` because it's already configured in our `settings.yml`.
```
Comment (jamespohalloran) [inline readme.md:234]: I think this section is still a bit confusing

**Review by jamespohalloran — CHANGES_REQUESTED** [review]
I nitpicked on some of the wording a bit here. It might be easier to review in the future with smaller sets of changes in separate PRs. But seems like good changes overall! I think the "table of contents" & screenshots will be especially helpful.

File: content/pages/home.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 ---
-title: Page Title
+title: My website
 blocks:
   - template: block-hero
-    heading: Welcome to my website!
-    message: This is some quality content right here. Here's another bit of text.
+    heading: "\U0001F44B Hi there!"
+    message: I'm testing editing a Next.js website in Tina
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply content/pages/home.md:6]: yes but unrelated.

File: readme.md (hunk 2 above)
Comment (DirtyF) [reply readme.md:234]: What's the point of specifying it's the relative path here?

**Comment by DirtyF** [comment]
@jamespohalloran I tried to address most concerns, let me know if anything is still blocking.

**Comment by jamespohalloran** [comment]
@DirtyF Cool I'll try and take a look later today. I think there are some comments that might have gotten missed though (Github condenses them in the UI)

File: readme.md [hunk 4]
```diff
@@ -1,138 +1,221 @@
...
 ```
 
-Make sure `NEXT_PUBLIC_USE_LOCAL_CLIENT` is set to `1`, other values can be ignored for now.
+## Edit content locally
 
-Restart your server and visit `http://localhost:3000/admin`, you should see the same content only this time there will be a Tina sidebar with fields you can edit and see live on the page. Saving a form here will result in changes to your local filesystem.
+We need to define some local environment variables in order to run the project.
 
-Read the [folder structure](#folder-structure) section below to learn more about how this site's routing works.
+Rename `.env.local.sample` to `.env.local`:
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Fixed.

File: readme.md [hunk 5]
```diff
@@ -1,138 +1,221 @@
...
+
+Basic TinaCMS starter based on [Next.js](https://nextjs.org) and [TinaCMS](https://tina.io) that connects to Tina Cloud Content API.
+
+![Starter Homepage Preview](public/uploads/tina-cloud-starter-preview.png)
+
+## Project scope:
+
+- Run this project locally to work with GitHub and the filesystem.
+- Connect to Tina Cloud to benefit from its GraphQL Content API.
+- Deploy the site to be able to invite others to visually edit your site 🔜.
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Fixed.

File: readme.md [hunk 6]
```diff
@@ -1,138 +1,221 @@
...
+- 🌎 [Run the project locally](#run-the-project-locally)
+- 📝 [Edit content locally](#edit-content-locally)
+- 🦙 [Connect to Tina Cloud](#connect-to-tina-cloud)
+    - ☁️ [Register your local application with Tina Cloud](#register-your-local-application-with-tina-cloud)
+    - 🔌 [Connect your local project with Tina Cloud](#connect-your-local-project-with-tina-cloud)
+    - 📝 [Edit content](#edit-content)
+- ⬆️ [Deploy](#deploy)
+    - ▲ [On Vercel](#on-vercel)
+    - [On Netlify](#on-netlify)
+- [Starter structure](#starter-structure)
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Fixed, Removed subheadings.

File: readme.md [hunk 7]
```diff
@@ -1,138 +1,212 @@
...
+`NEXT_PUBLIC_USE_LOCAL_CLIENT` should be set to `1`, other values can be ignored for now.
 
-### Continuing local development
+Restart your server and visit [`http://localhost:3000/admin`](http://localhost:3000/admin`), the same page is displayed but you can notice a pencil at the bootom left corner.
 
-While it's pretty quick to get started, there's a lot more to show on how you can work locally, read the [workflow tips](#local-development-workflow-tips) for more info and check out the [development walkthrough](https://www.loom.com/share/e62776f138ec485d81d71c68364857a8) video.
+Click to open Tina sidebar which displays a form with fields you can edit and update live on the page. Since we're working locally, saving results in changes to your local filesystem.
 
-## Connecting to Tina Cloud
+> 🐛 issue: when adding a block and populating its content, that data on your page updates to the wrong block.[Watch this issue](https://github.com/tinacms/tinacms/issues/1669) or maybe by us.
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Fixed.

File: readme.md [hunk 8]
```diff
@@ -1,138 +1,212 @@
...
 
-While the fully-local development workflow is the recommended way for developers to work, you'll obviously want other editors and collaborators to be able to make changes on a hosted website with authentication. In general it's a good idea to avoid working locally while communicating with the Tina Cloud API, but it's something you'll want to test to ensure it works as expected.
+Read the [folder structure](#folder-structure) section below to learn more about how this site's routing works.
 
-> Note that changes to cloud content will only be shown in the `/admin` route. This is because the repo is designed to source content from your filesystem and build it statically during deployments.
+## Connect to Tina Cloud
 
-### Register your local app with Tina Cloud
+While the fully-local development workflow is the recommended way for developers to work, you'll obviously want other editors and collaborators to be able to make changes on a hosted website with authentication. We recommend to avoid working locally while communicating with the Tina Cloud API, even if it's something you need to test to ensure it works as expected.
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: I just moved this part. @jeffsee55 Can we remove it now?

File: readme.md [hunk 9]
```diff
@@ -1,138 +1,221 @@
...
+- [Starter structure](#starter-structure)
+    - 🗂 [Folder structure](#folder-structure)
+    - 📄 [pages/\[\[...slug\]\].tsx](#pagesslugtsx)
+    - [Relative Path](#relative-path)
+    - [pages/admin/\[\[...slug\]\].tsx](#pagesadminslugtsx)
+    - [components/document-renderer.tsx](#componentsdocument-renderertsx)
+- 💡 [Local development workflow tips](#local-development-workflow-tips)
+    - 👀 [Watch our Development Walkthrough](#watch-our-development-walkthrough)
+    - [Typescript](#typescript)
+    - [Visual Studio Code](#visual-studio-code)
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Removed.

File: readme.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: @jamespohalloran I removed the last sentence.

File: readme.md [hunk 10]
```diff
@@ -1,138 +1,212 @@
...
+Read the [folder structure](#folder-structure) section below to learn more about how this site's routing works.
 
-> Note that changes to cloud content will only be shown in the `/admin` route. This is because the repo is designed to source content from your filesystem and build it statically during deployments.
+## Connect to Tina Cloud
 
-### Register your local app with Tina Cloud
+While the fully-local development workflow is the recommended way for developers to work, you'll obviously want other editors and collaborators to be able to make changes on a hosted website with authentication. We recommend to avoid working locally while communicating with the Tina Cloud API, even if it's something you need to test to ensure it works as expected.
 
-Visit [auth.tinajs.dev](https://auth.tinajs.dev/register), create a realm, and sign in.
+> ℹ️ Changes to cloud content will only be shown in the `/admin` route. This is because the repository is designed to source content from your filesystem and build it statically during deployments.
```
[the remark this answers is not in the document]
Comment (DirtyF) [reply readme.md]: Fixed.

File: readme.md [hunk 11]
```diff
@@ -1,138 +1,212 @@
...
+Changes are saved in your GitHub repository.
 
-These changes will be persisted to Github. Note that you won't see them when you visit a non-admin route when working locally.
+> Note: These changes are not reflected when you visit a non-admin route when working locally.
 
-# Hosting the project
+## Deploy
 
-At this point you have Tina Cloud editing enabled, deploy it to the cloud so others can make edits too:
+Once Tina Cloud editing is working, you must deploy it so others can make edits too.
```
[the remark this answers is not in the document]
Comment (jamespohalloran) [reply readme.md]: I don't think this got changed before marking as resolved. My issues with this sentence were that it sounds weird to say that "you must deploy the site...". Also, I could see users not being sure how to measure: "Once Tina Cloud editing is working"

File: readme.md [hunk 12]
```diff
@@ -1,138 +1,212 @@
...
 
-## Hosting on Vercel
+### On Vercel
```
[the remark this answers is not in the document]
Comment (jamespohalloran) [reply readme.md]: To me, the headings "On Vercel", "On Netlify" feel weird on their own.

File: readme.md [hunk 13]
```diff
@@ -1,138 +1,212 @@
...
-_You will need to trigger a redeploy from Vercel's UI for these environment variables to take effect_
-
-Now that we have a live site, we can take the deployment URL and use it within our Tina Cloud app's `Callback URL` field.
-Go to the [dashboard](https://auth.tinajs.dev/register), click into your new app, and change its `Callback URL` to `[your deployment URL]/admin`
+![](public/uploads/vercel-congratulations.png)
 
-You can test that everything is configured correctly by navigating to `[your deployment URL]/admin`, and trying to login.
+🎉 Congratulations, your site is now live! \
+\
+You can test that everything is configured correctly by navigating to `[your deployment URL]/admin`, login to Tina Cloud and doing some edits that are saved to your GitHub repository.
```
[the remark this answers is not in the document]
Comment (jamespohalloran) [reply readme.md]: I think this was missed

File: readme.md [hunk 14]
```diff
@@ -1,138 +1,212 @@
...
-NEXT_PUBLIC_TINA_CLIENT_ID=<get this from the app you create at auth.tinajs.dev>
+NEXT_PUBLIC_REALM_NAME= <YOUR_REALM>
+NEXT_PUBLIC_TINA_CLIENT_ID= <YOUR_CLIENT_ID>
 ```
 
-_You will need to trigger a redeploy from Netlify's UI for these environment variables to take effect_
+Set the **build command** to `yarn build`,
+Set the **publish directory**. To `.next/` .
+
+Once you're done, click deploy site.
```
[the remark this answers is not in the document]
Comment (jamespohalloran) [reply readme.md]: missed this one

File: readme.md [hunk 15]
```diff
@@ -1,138 +1,212 @@
...
+
+Install the ["Next on Netlify" plugin](https://www.netlify.com/blog/2020/12/07/announcing-one-click-install-next.js-build-plugin-on-netlify/) in order to take advantage of server-side rendering and Next.js preview features.
 
-Now that we have a live site, we can take the deployment URL and use it within our Tina Cloud app's `Callback URL` field.
-Go to the [dashboard](https://auth.tinajs.dev), click into your new app, and change its `Callback URL` to `[your deployment URL]/admin`
+Trigger a new deploy for changes to take effect.
 
-You can test that everything is configured correctly by navigating to `[your deployment URL]/admin`, and trying to login.
+Test that everything is configured correctly by navigating to `[your deployment URL]/admin`, login, and doing some edits.
```
[the remark this answers is not in the document]
Comment (jamespohalloran) [reply readme.md]: missed this one

**Review by jamespohalloran — APPROVED** [review]

**Pull request merged**